### DISCUSSION TITLE
Relevant items to boost crops

### DISCUSSION THREAD

**Discussion by Yougottabekiddingme123** [sunflower-land/sunflower-land#1363] [General]
How does an amulet boost crops? This doesn't make any sense from a logical or lore perspective.

I'm sure the community can come with with more logical and relevant items to provide boosts to crops.

Of the top of my head, I would say:
Farmers Gloves
Water Spray
Pruning tool

I find that some of the items are just copying pervasive nft items in the RPG  cryptosphere instead of creating items that are relevant to farming and SFL.

Please ask the community for feedback and suggestions for items. You decide on the boosts that the item provides but let the community contribute and provide feedback/suggestions for items.

**Comment by AL-Simmons92** [comment]
Goblins, Mutants, Kings and Queens..... there is fantasy in the game.... amulets of power are a thing. They can add  lore to these kinda things. a pair of gloves.... boring. Just my .02 Farmer Bill Out......
